user_docs/en/userGuide.t2t

```diff
@@ -3511,88 +3511,170 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
+| ``PageDown`` key | dot4+dot6+``space`` |
+| ``Numpad1`` key | dot1+dot6+``backspace`` |
+| ``Numpad2`` key | dot1+dot2+dot6+``backspace`` |
+| ``Numpad3`` key | dot1+dot4+dot6+``backspace`` |
+| ``Numpad4`` key | dot1+dot4+dot5+dot6+``backspace`` |
+| ``Numpad5`` key | dot1+dot5+dot6+``backspace`` |
+| ``Numpad6`` key | dot1+dot2+dot4+dot6+``backspace`` |
+| ``Numpad7`` key | dot1+dot2+dot4+dot5+dot6+``backspace`` |
+| ``Numpad8`` key | dot1+dot2+dot5+dot6+``backspace`` |
+| ``Numpad9`` key | dot2+dot4+dot6+``backspace`` |
+| ``NumpadInsert`` key | dot3+dot4+dot5+dot6+``backspace`` |
+| ``NumpadDecimal`` key | dot2+``backspace`` |
+| ``NumpadDivide`` key | dot3+dot4+``backspace`` |
+| ``NumpadMultiply`` key | dot3+dot5+``backspace`` |
+| ``NumpadMinus`` key | dot3+dot6+``backspace`` |
+| ``NumpadPlus`` key | dot2+dot3+dot5+``backspace`` |
+| ``NumpadEnter`` key | dot3+dot4+dot5+``backspace`` |
+| ``Escape`` key | dot1+dot2+dot4+dot5+``space``, l2 |
+| ``Tab`` key | dot2+dot5+dot6+``space``, l3 |
+| ``Shift+tab`` key | dot2+dot3+dot5+``space`` |
+| ``PrintScreen`` key | dot1+dot3+dot4+dot6+``space`` |
+| ``Pause`` key | dot1+dot4+``space`` |
+| ``Applications`` key | dot5+dot6+``backspace`` |
+| ``F1`` key | dot1+``backspace`` |
+| ``F2`` key | dot1+dot2+``backspace`` |
+| ``F3`` key | dot1+dot4+``backspace`` |
+| ``F4`` key | dot1+dot4+dot5+``backspace`` |
+| ``F5`` key | dot1+dot5+``backspace`` |
+| ``F6`` key | dot1+dot2+dot4+``backspace`` |
+| ``F7`` key | dot1+dot2+dot4+dot5+``backspace`` |
+| ``F8`` key | dot1+dot2+dot5+``backspace`` |
+| ``F9`` key | dot2+dot4+``backspace`` |
+| ``F10`` key | dot2+dot4+dot5+``backspace`` |
+| ``F11`` key | dot1+dot3+``backspace`` |
+| ``F12`` key | dot1+dot2+dot3+backspace`` |
+| ``Windows`` key | dot1+dot2+dot4+dot5+dot6+``space`` |
+| Toggle ``windows`` key | dot1+dot2+dot3+dot4+``backspace``, dot2+dot4+dot5+dot6+``space`` |
+| ``Caps lock`` key | dot7+``backspace``, dot8+``backspace`` |
+| ``Num lock`` key | dot3+``backspace``, dot6+``backspace`` |
+| ``Shift`` key | dot7+``space`` |
+| Toggle ``shift`` key | dot1+dot7+``space``, dot4+dot7+``space`` |
+| ``Control`` key | dot7+dot8+``space`` |
+| Toggle ``control`` key | dot1+dot7+dot8+``space``, dot4+dot7+dot8+``space`` |
+| ``Alt`` key | dot8+``space`` |
+| Toggle ``alt`` key | dot1+dot8+space, dot4+dot8+``space`` |
+%kc:endInclude
+
++++ b.book keyboard commands +++[Eurobraillebbook]
+%kc:beginInclude
+|| Name | Key |
+| Scroll braille display back | backward |
+| Scroll braille display forward | Forward |
+| Move to current focus | Backward + forward | 
+| Route to braille cell | routing |
+| Scroll braille display back | backward |
+| Scroll braille display forward | Forward |
+| Move to current focus | Backward + forward |
+| Route to braille cell | routing |
+| Scroll braille display back | backward |
+| Scroll braille display forward | Forward |
+| Move to current focus | Backward + forward |
+| Route to braille cell | routing |

```

Comment:
these are duplicated three times, is there a reason for it?

Reply:
That's a mistake I made while copying the provided documentation from Eurobraille to the t2t format. Thank you for noticing it!